### DISCUSSION TITLE
Adding contributors

### DISCUSSION THREAD

**Discussion by bob420-svg** [community/community#65666] [Pull Requests] [answered]
How do I add team members to comments? I think that's what it meant. Said to add team members in the pull request comments

**Comment by tuves** [comment]
Welcome to the GitHub Community, @cghost420, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by bob420-svg** [reply, marked as answer]
Thank you! Sorry, I'm trying to figure this out.
